test/acceptance/controller.js

```diff
@@ -462,6 +463,9 @@ describe('Controller', function () {
         client
           .get('/jpg/50/0/0/801/478/0/0/0/2/aspectfit/North/0/0/0/0/0/testxxx.jpg')
           .end(function (err, res) {
+            let isBuffer = (res.body instanceof Buffer)
```

Comment:
Could this be just `res.body.should.be.instanceof(Buffer)`? 🤔

Reply:
I knew there was a better way!